ctms/models.py

```diff
@@ -249,6 +249,8 @@ class PendingAcousticRecord(Base, TimestampMixin):
 
     email = relationship("Email", uselist=False)
 
+    __table_args__ = (UniqueConstraint("email_id", name="uix_pr_email_id"),)
```

Comment:
I think it's the same end result, but can we keep things consistent and use `unique=True` on the `email_id` `Column` above? It looks like we reach for the `UniqueConstraint` when it's a composite index like `UniqueConstraint("email_id", "name", name="uix_email_name")`